__mocks__/react_pconnect.jsx

```diff
@@ -14,36 +14,51 @@ let detailsIndex = 0;
 let detailsRegionIndex = 0;
 
 const createPConnectComponent = () => {
-  const formComponents = [TextInput, TextInput,TextInput, Phone, AutoComplete, Email];
+  const formComponents = [TextInput, TextInput, TextInput, Phone, AutoComplete, Email];
   const detailsComponents = [TextInput, TextInput, TextInput, Email, Phone, Currency, RadioButtons, Operator];
-  const detailsRegionComponents = [TextInput, TextInput, TextInput, Email, Phone, Currency, RadioButtons, TextInput, TextInput, TextInput, Email, Phone, Currency, Operator];
+  const detailsRegionComponents = [
+    TextInput,
```

Comment:
Components are repeated multiple times

Reply:
@ghoshArnab This is intentional as part of moacking createPConnectComponent API.